go/sqltypes/value.go

```diff
@@ -548,7 +548,12 @@ func encodeBytesSQL(val []byte, b BinWriter) {
 
 func encodeBytesSQLBytes2(val []byte, buf *bytes2.Buffer) {
 	buf.WriteByte('\'')
-	for _, ch := range val {
+	for idx, ch := range val {
+		// If \% or \_ is present, we want to keep them as is, and don't want to escape \ again
+		if ch == '\\' && idx+1 < len(val) && (val[idx+1] == '%' || val[idx+1] == '_') {
```

Comment:
is '%' and '_' the only ones that need this change or there are others as well?

Reply:
These are the only ones. I'll caveat by adding that these are the only ones according to the MySQL documentation! 😆